recipes/bdbag/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "bdbag" %}
+{% set version = "1.5.6" %}
+{% set hash_type = "md5" %}
+{% set hash_value = "66a6878203aebc055cef6683322d2f73" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_value }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - bdbag = bdbag.bdbag_cli:main
+    - bdbag-utils = bdbag.bdbag_utils:main
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - pytz
+    - tzlocal
+    - requests
+    - certifi
+    - bagit
+    - boto3 >=1.9.5
+    - botocore
+    - awscli
+    - globus-sdk >=1.6.0
+
+test:
+  imports:
+    - bdbag
+  commands:
+    bdbag -h
+
+about:
+  home: https://github.com/fair-research/bdbag
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE.txt
```

Comment:
The tarball seems to contain the `LICENSE`. I don't think you need to ship this explicitly with the recipe.

Reply:
Apparently it needs to be there or the linting check fails.

Reply:
Right. Needs to be in that section but you don't need to package the file. I believe the file is called `LICENSE` in the tarball. So, just calling it `LICENSE` should pick up the file from the tarball.